llvm/include/llvm/Analysis/ScalarEvolution.h

```diff
@@ -1167,6 +1207,7 @@ class ScalarEvolution {
   /// If \p AllowPredicates is set, this call will try to use a minimal set of
   /// SCEV predicates in order to return an exact answer.
   ExitLimit computeExitLimitFromCond(const Loop *L, Value *ExitCond,
+                                     std::function<LoopGuards()> GetLoopGuards,
```

Comment:
std::function -> function_ref

Reply:
Also shouldn't this be `const LoopGuards &()`? Otherwise you're going to return a copy of the loop guards...

Reply:
Argh, that's probably it. Will fix...